environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
There are feature changes between 7.6 and 7.5 because the v8 version has been bumped up. I now have them grouped on node.green and it's easier to see when v8 changes cause feature changes.